app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] @team great work on your devise setup however you forgot to add `:confirmable` in your user model to allow the application to ask for confirmation of the email. This is a project requirement, kindly add the functionality.
![image](https://user-images.githubusercontent.com/61408860/195423655-e691082a-c9df-492d-b970-e557b2663efa.png)

Reply:
Already added Confirmable according to previous review.

Reply:
@DarioAlessioR that all that was required, and sorry for the ate response